modules/ocf_hpc/facts.d/nvidia_gpu_info

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import json
+import re
+import shutil
+import subprocess
+
+
+def main():
+    if not shutil.which('nvidia-smi'):
+        return
+
+    try:
+        nvidia_smi_output = subprocess.run(['nvidia-smi', '--query-gpu=name,index', '--format=csv,noheader'],
```

Comment:
I'd also use `subprocess.check_output` here, like I mentioned in #351, I think it'll clean things up a fair amount.